R/staticimports.R

```diff
@@ -0,0 +1,12 @@
+# Generated by staticimports; do not edit by hand.
+# ======================================================================
+# Imported from pkg:staticimports
+# ======================================================================
+
+is_installed <- function(package, version = NULL) {
+  installed <- nzchar(system.file(package = package))
```

Comment:
Pulling in latest from https://github.com/wch/staticimports/pull/11/files#diff-a73e22a3c06b2dfbaa85dc6a4b2478afececce51dce97b2b95c40570fb5632dcR6
```suggestion
  installed <- isNamespaceLoaded(package) || nzchar(system.file(package = package))
```